Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
OK. so after a bit more search, here's the relevant parts of a question and its [answer](https://groups.google.com/forum/embed/#!topic/fpm-users/ra9ifNkX8s8) by Jordan Sissel (author):

> **question:** if I want to change the permissions of the files being installed with a deb file what is the best approach for this?
> **answer:** use `--deb-user` and `--deb-group` to force user/group on all files in the package; use `chmod` prior to packaging to set permissions (but probably ownership)

fyi, per the docs, if you don't specify `--deb-user` and `--deb-group`, default is `root` and `root` respectively. still confused about the usage of `--deb-use-file-permissions`, so opened an issue here: https://github.com/jordansissel/fpm/issues/1171